src/components/KSelect/KSelect.vue

```diff
@@ -238,435 +255,407 @@ export interface SelectFilterFnParams {
 
 export type DropdownFooterTextPosition = 'sticky' | 'static'
 
-export default defineComponent({
-  name: 'KSelect',
-  components: {
-    KButton,
-    KIcon,
-    KInput,
-    KLabel,
-    KPop,
-    KSelectItems,
-    KSelectItem,
-    KToggle,
+</script>
+
+<script setup lang="ts">
```

Comment:
**Note for PR review:** all changes in `<script>` tag are due to switch to `script setup` syntax (this PR doesn't introduce any changes in logic).